chart/values.yaml

```diff
@@ -427,65 +378,94 @@ ingress:
+    extraEnv: []
+    image:
+      repository: docker.io/traefik
+      tag: v3.5.3
+  replicaCount: 1
+  updateStrategy: ~
+  podLabels: {}
+  configmapAnnotations: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
 
 # subchart settings
-postgresql:
+postgres:
   enabled: false
   auth:
     # default user to be created by postgres subchart
     # subchart is unable to consume a secret for specifying user
-    username: kyoo_all
+    username: "{{ .Values.global.postgres.infra.user }}"
     existingSecret: "{{ .Values.global.postgres.infra.existingSecret }}"
     secretKeys:
       # set the postgres user password to the same as our user
-      adminPasswordKey: "{{ .Values.global.postgres.infra.passwordKey }}"
-      userPasswordKey: "{{ .Values.global.postgres.infra.passwordKey }}"
-  primary:
-    # create databases, schemas, and set search_path
-    initdb:
-      scripts:
-        # kyoo_back still requires public schema
-        # https://github.com/zoriya/Kyoo/issues/536
-        kyoo_back.sql: |
-          CREATE DATABASE {{ .Values.global.postgres.kyoo_back.database }} WITH OWNER {{ .Values.global.postgres.infra.user }}; 
-          \connect {{ .Values.global.postgres.kyoo_back.database }};
-          CREATE SCHEMA IF NOT EXISTS data AUTHORIZATION {{ .Values.global.postgres.infra.user }};
-        kyoo_transcoder.sql: |
-          CREATE DATABASE {{ .Values.global.postgres.kyoo_transcoder.database }} WITH OWNER {{ .Values.global.postgres.infra.user }};
-          \connect {{ .Values.global.postgres.kyoo_transcoder.database }};
-          REVOKE ALL ON SCHEMA public FROM PUBLIC;
-          CREATE SCHEMA IF NOT EXISTS data AUTHORIZATION {{ .Values.global.postgres.infra.user }};
-        user.sql: |
-          ALTER ROLE {{ .Values.global.postgres.infra.user }} 
-          IN DATABASE {{ .Values.global.postgres.kyoo_back.database }} SET search_path TO "$user", public;
-          ALTER ROLE {{ .Values.global.postgres.infra.user }} 
-          IN DATABASE {{ .Values.global.postgres.kyoo_transcoder.database }} SET search_path TO "$user", data;
-    persistence:
-      size: 3Gi
-
-# subchart settings
-rabbitmq:
-  enabled: false
-  auth:
-    # default user to be created by rabbitmq subchart
-    # subchart is unable to consume a secret for specifying user
-    username: kyoo_all
-    existingPasswordSecret: "{{ .Values.global.rabbitmq.infra.existingSecret }}"
-    existingSecretPasswordKey: "{{ .Values.global.rabbitmq.infra.passwordKey }}"
-    existingErlangSecret: "{{ .Values.global.rabbitmq.infra.existingSecret }}"
-    existingSecretErlangKey: "{{ .Values.global.rabbitmq.infra.keyErlangCookie }}"
+      passwordKey: "{{ .Values.global.postgres.infra.passwordKey }}"
+  initdb:
```

Comment:
you cannot select a name for the schema services will use (they are hard coded).

here's the mapping:
```
api: kyoo
auth: keibi
scanner: scanner
transcoder: gocoder
```